src/components/Form/MoreOptions/index.js

```diff
@@ -27,6 +27,7 @@ export default function MoreOptions({ enabled, onClick }) {
         <div
             className={s.container}
             onClick={() => onClick(enabled ? CTX_DEFAULT : CTX_MORE_OPTIONS)}
+            data-test="more-options-button"
```

Comment:
Yeah no problem 👍. Just make a note in dhis2/notes#88 if you've encountered anything different from what we're recommending there, so that it's up to date.